lib/project/ext/ui_image_view.rb

```diff
@@ -1,6 +1,16 @@
 class UIImageView
 
-  def remote_image=(value)
+  def remote_image=(url)
+    load_remote_image(url)
+  end
+
+  def remote_image(args)
+    load_remote_image(args.fetch(:url), args.fetch(:did_load))
```

Comment:
Won't using `fetch` without specifying a second argument require that a `did_fetch` is specified? In other words, I think this should be `args.fetch(:did_load, nil)` to allow it to be optional.